django/contrib/admin/templates/admin/app_list.html

```diff
@@ -7,6 +7,13 @@
         <caption>
           <a href="{{ app.app_url }}" class="section" title="{% blocktranslate with name=app.name %}Models in the {{ name }} application{% endblocktranslate %}">{{ app.name }}</a>
         </caption>
+        <thead class="visually-hidden">
+          <tr>
+            <th scope="col">{% translate 'Model name' %}</th>
+            <th scope="col">{% translate 'Add link' %}</th>
+            <th scope="col">{% translate 'Change list link' %}</th>
```

Comment:
This can be also a `view`-only link so maybe 
```suggestion
            <th scope="col">{% translate 'Change or view list link' %}</th>
```